cmd/kueue/main.go

```diff
@@ -130,7 +130,9 @@ func main() {
 		os.Exit(1)
 	}
 
-	metrics.Register()
```

Comment:
why is `metrics.Register()` gone? I would suggest to move`metrics.RegisterLQMetrics()` inside `metrics.Register()`

Reply:
I think this was likely a bug which would cause no metrics at all, so I would like to have some e2e tests for metrics (separate PR)